### PR TITLE
pkp/pkp-lib#8344 Added country validation for contributor api

### PR DESCRIPTION
**Pull request by touhidurabir**
This PR aims to solve pkp/pkp-lib#8344

### CODE REVIEW

File: locale/en_US/common.po [hunk 1]
```diff
@@ -1946,6 +1946,9 @@ msgstr "The confirmation for this field does not match."
 msgid "validator.country"
 msgstr "This is not a valid country."
 
+msgid "validator.country.regex"
+msgstr "Invalid country code formate. Country code must be a 2 letter code in english capital letter."
```
Comment (NateWr) [inline locale/en_US/common.po:1950]: > This is not a valid country. The country must be one of the 2-letter international codes. See all codes at https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements.

File: schemas/author.json [hunk 2]
```diff
@@ -26,7 +26,14 @@
 			]
 		},
 		"country": {
-			"type": "string"
+			"type": "string",
+			"description": "ISO 3166-1 two letter code in english capital letter.",
```
Comment (NateWr) [inline schemas/author.json:30]:
```
The author's country in a two-letter code (ISO 3166-1). See [all codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements).
```

(markdown in the description is rendered correctly in the resulting API documentation)

**Review by NateWr — CHANGES_REQUESTED** [review]
Looks great, @touhidurabir. Just a couple of suggested changes to the text descriptions. You don't need to re-run the tests. Once you've made the changes let me know and I'll merge.

File: locale/en_US/common.po (hunk 1 above)
Comment (touhidurabir) [reply locale/en_US/common.po]: Done.

File: schemas/author.json (hunk 2 above)
Comment (touhidurabir) [reply schemas/author.json]: done.

**Pull request merged**